lib/widgets/CustomButton/custom_button.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:classroom_app/styles/color/colors.dart';
+
+class CustomButton extends StatelessWidget {
+  const CustomButton({
+    super.key,
+    required this.buttonText,
+    required this.buttonBgColor,
+    this.buttonTextColor,
+    this.buttonBorderColor,
+  });
+
+  final String buttonText;
+  final Color buttonBgColor;
```

Comment:
```suggestion
  final Color? buttonBgColor;
```

Reply:
Every button does have a background color and must have to pass the color. That's why it was made not nullable.